pkg/volume/iscsi/iscsi_util.go

```diff
@@ -872,8 +877,14 @@ func cloneIface(b iscsiDiskMounter, newIface string) error {
 	// create new iface
 	out, err = b.exec.Run("iscsiadm", "-m", "iface", "-I", newIface, "-o", "new")
 	if err != nil {
-		lastErr = fmt.Errorf("iscsi: failed to create new iface: %s (%v)", string(out), err)
-		return lastErr
+		if exit, ok := err.(utilexec.ExitError); ok {
+			if exit.ExitStatus() == iscsiadmErrorSessExists {
+				klog.Infof("iscsi: there is a session already logged in with iface %s", newIface)
+			}
```

Comment:
If the case to ExitError succeeds, but the error code doesn't match, this won't return an error like it should.

Reply:
Fixed.